app/assets/javascripts/admin/customers/services/customers.js.coffee

```diff
@@ -25,3 +25,11 @@ angular.module("admin.customers").factory "Customers", ($q, InfoDialog, RequestM
       request = CustomerResource.index(params, (data) => @customers = data)
       RequestMonitor.load(request.$promise)
       request.$promise
+
+    update: (address, customer, current_address) ->
```

Comment:
Which values does `current_address` have? I guess `bill_address` or `ship_address`. In that case we could name the variable `address_type` or similar. Because when reading `current_address`, I expect it to contain a value like "123 Hight St, Thornbury Victoria".